conans/test/functional/preexport_test.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+
+from conans.test.utils.tools import TestClient
+
+
+class PreExportTest(unittest.TestCase):
+
+    def load_from_file_test(self):
+        client = TestClient()
+        conanfile = """from conans import ConanFile, load
+import os
+class Pkg(ConanFile):
+    name = "MyPkg"
+
+    def preexport(self):
```

Comment:
Syntax ``preexport()`` seems almost like a typo. `pre-export()`, `pre_export()`... whichever option looks better for me. But, is this really intended for "pre export" usage only or to load information even for local commands?

Reply:
Yeah, agree, the name can be changed, maybe ``pre_export()`` is ok. ``pre-export()`` is not valid python.